src/cost_estimates/pessimistic.rs

```diff
@@ -230,6 +230,7 @@ impl PessimisticEstimator {
                     StacksEpochId::Epoch20 => "",
                     StacksEpochId::Epoch2_05 => ":2.05",
                     StacksEpochId::Epoch21 => ":2.1",
+                    StacksEpochId::Epoch22 => ":2.2",
```

Comment:
hmmm... do we need to "purge" estimate data if the cost computation does not change? perhaps we should create these keys differently... just a thought though.

Reply:
Addressed in 06e546951.